2.cellprofiler_processing/README.md

```diff
@@ -77,7 +77,7 @@ To run CellProfiler through the command line, run the [example_cp.ipynb notebook
 bash cellprofiler_feature_seg.sh
 ```
 
-CellProfiler runs on CPU, and it took approximately 9 hours and 50 minutes to run plate `localhost220512140003_KK22-05-198` and approximately 5 hours to run plate `localhost220513100001_KK22-05-198_FactinAdjusted`. 
+CellProfiler runs on CPU, and it took approximately 10 hours to run plate `localhost220512140003_KK22-05-198` and approximately 5 hours to run plate `localhost220513100001_KK22-05-198_FactinAdjusted`. 
```

Comment:
Noticing that this line changes every so often, it might be helpful to consider using a library or module to assist with this so it's automatically embedded (and referenceable here other readme's). Jupyter's [`%time`](https://ipython.readthedocs.io/en/stable/interactive/magics.html#magic-time) or using shell `$SECONDS` creatively (for example, see this [SO answer](https://stackoverflow.com/a/5153036)).

Reply:
Not sure how this would work, but definitely seems like something that can be implemented! I will discuss this with you!